charts/model-engine/values.yaml

```diff
@@ -16,3 +16,5 @@ balloonNodeSelector:
   node-lifecycle: normal
 nodeSelector:
   node-lifecycle: normal
+keda:
```

Comment:
can you also add it to value_samples and everywhere else, otherwise please set default values

Reply:
is there a list of all files for where the update should be made? I based my logic of this [PR](https://github.com/scaleapi/llm-engine/pull/651) which seems to only update one values file

Reply:
actually i think it might be fine. helm should be able to take default values from values.yaml

Reply:
ok great, will merge then. appreciate the review!